packages/conventional-changelog-cli/test/test.js

```diff
@@ -21,15 +17,13 @@ function originalChangelog () {
 
 describe('cli', function () {
   before(function () {
-    shell.config.resetForTesting()
-    shell.cd(__dirname)
-    shell.rm('-rf', 'tmp')
-    shell.mkdir('tmp')
-    shell.cd('tmp')
-    shell.mkdir('git-templates')
-    shell.exec('git init --template=./git-templates')
+    process.chdir(__dirname)
+    rimraf.sync('tmp')
```

Comment:
`rimraf` is added as replacement for `rm -rf` as our tested node version does not support `recursive` option in `rmDirSync`